.rubocop.yml

```diff
@@ -13,7 +13,25 @@ Metrics/BlockLength:
 ModuleLength:
   Enabled: false
 
-Style/EmptyMethod:
+Style/GlobalVars:
+  Enabled: false
+
+Style/IndentArray:
+  Enabled: false
+
+Style/AlignHash:
+  Enabled: false
+
+Style/Lambda:
```

Comment:
IMHO it is better to use one kind of lambda syntax (literal) in the whole project. Can be fixed automatically.